app/graphql/types/host.rb

```diff
@@ -4,9 +4,19 @@ class Host < BaseObject
 
     global_id_field :id
     timestamps
-    field :name, String, null: false
+    field :name, String, null: true
+    field :build, Boolean, null: true
+    field :ip, String, null: true
+    field :ip6, String, null: true
+    field :path, resolver: Resolvers::Host::Path
```

Comment:
Can we also add `mac`, `last_report`, `domain_name`, `pxe_loader`, `enabled`, `uuid` here? They are low hanging fruit.

Reply:
added